src/azure-cli/azure/cli/command_modules/storage/sdkutil.py

```diff
@@ -25,9 +25,9 @@ def get_table_data_type(cli_ctx, module_name, *type_names):
 
 def get_blob_service_by_type(cli_ctx, blob_type):
     type_to_service = {
-        'block': lambda ctx: get_sdk(ctx, ResourceType.DATA_STORAGE, 'BlockBlobService', mod='blob'),
-        'page': lambda ctx: get_sdk(ctx, ResourceType.DATA_STORAGE, 'PageBlobService', mod='blob'),
-        'append': lambda ctx: get_sdk(ctx, ResourceType.DATA_STORAGE, 'AppendBlobService', mod='blob')
+        'block': lambda ctx: get_sdk(ctx, ResourceType.DATA_STORAGE, 'BlockBlobService', mod='blob', checked=False),
+        'page': lambda ctx: get_sdk(ctx, ResourceType.DATA_STORAGE, 'PageBlobService', mod='blob', checked=False),
+        'append': lambda ctx: get_sdk(ctx, ResourceType.DATA_STORAGE, 'AppendBlobService', mod='blob', checked=False)
```

Comment:
I would really prefer to make `checked=False` the default, otherwise it is not universal and `'NoneType' object has no attribute '__name__'` will continue to happen in other command modules. (https://github.com/Azure/azure-cli/pull/15776#discussion_r520283336)

Reply:
I already made `checked=False` the default for `_get_attr` in #14690, but it only applies to `get_client_class` which calls `_get_attr` without `checked`:

https://github.com/Azure/azure-cli/blob/301ba06ff13c354b4499fd18913d294a7926aef6/src/azure-cli-core/azure/cli/core/profiles/_shared.py#L552-L553

`get_versioned_sdk` will default `checked` to `True` via

https://github.com/Azure/azure-cli/blob/301ba06ff13c354b4499fd18913d294a7926aef6/src/azure-cli-core/azure/cli/core/profiles/_shared.py#L573-L574

https://github.com/Azure/azure-cli/blob/301ba06ff13c354b4499fd18913d294a7926aef6/src/azure-cli-core/azure/cli/core/profiles/_shared.py#L585

Reply:
I was afraid of making great changes to all modules. Should we discuss whether to change the behavior or not with all module owners?

Reply:
We can go ahead and make the change. If CI passes, we are fine.

Reply:
Python's `import` statement raises [`ImportError`](https://docs.python.org/3/library/exceptions.html#ImportError) in case of non-existing module.

Also, checking whether the result is `None` is against the [look before you leap](https://docs.python.org/3/glossary.html#term-lbyl) practice.